packages/framework/framework/package.json

```diff
@@ -42,25 +48,31 @@
   "author": "Medusa",
   "license": "MIT",
   "scripts": {
-    "watch": "tsc --build --watch -p ./tsconfig.build.json",
+    "watch": "tsc --watch -p ./tsconfig.build.json",
     "watch:test": "tsc --build tsconfig.spec.json --watch",
     "prepublishOnly": "cross-env NODE_ENV=production tsc -p ./tsconfig.build.json && tsc-alias -p ./tsconfig.build.json",
     "build": "rimraf dist && tsc --build  && tsc-alias",
     "test": "jest --runInBand --bail --passWithNoTests --forceExit -- src",
     "test:integration": "jest --forceExit -- integration-tests/**/__tests__/**/*.ts"
   },
   "devDependencies": {
-    "@medusajs/types": "^1.11.17-preview-20240510084332",
+    "@medusajs/types": "^1.11.16",
+    "@types/express": "^4.17.17",
     "cross-env": "^7.0.3",
-    "ioredis": "^5.2.5",
     "rimraf": "^3.0.2",
     "tsc-alias": "^1.8.6",
     "typescript": "^5.1.6",
     "vite": "^5.2.11"
   },
   "dependencies": {
-    "@medusajs/medusa-cli": "^1.3.23",
-    "@medusajs/utils": "^1.12.0-preview-20240724081425",
-    "awilix": "^8.0.0"
+    "@medusajs/medusa-cli": "^1.3.22",
+    "@medusajs/utils": "^1.12.0-snapshot-20240728140928",
```

Comment:
@olivermrbl should i move that back to preview before we merge?

Reply:
Can you move them to use their latest versions with a caret (`^`). I think then we can merge

Reply:
let me try, last time it was failing for me 😂

Reply:
Looks like this breaks everything 🤒

Reply:
let me try something

Reply:
The the wrong version of utils is installed right now. There is no 1.11.10

Reply:
Yes there is one https://www.npmjs.com/package/@medusajs/utils/v/1.11.10

Reply:
Otherwise the i stall would have failed with 404 and not reaching the build

Reply:
Now I have another question, how come 1.11.10 does not have the latest changes when it is the latest version and i have to downgrade ?

Reply:
Yes, I noticed this. Edited my comment :)

Reply:
> Now I have another question, how come 1.11.10 does not have the latest changes when it is the latest version and i have to downgrade ?

This is was spawned my comment initially. I was wondering this too. My immediate thought was that this is because 1.11.10 has been released as part of v1.x, and that branch doesn't have everything from `develop`.

Reply:
ahah makes sense 👍

Reply:
Pretty sure this is actually what has been causing issues all along.

Reply:
Yes probably, and i ve missed it for so long 😣

Reply:
Let's slap an automerge on this, shall we?

Reply:
done 😍